### DISCUSSION TITLE
How to test tRPC on Thunder Client or Postman

### DISCUSSION THREAD

**Discussion by jpmadrigal07** [trpc/trpc#2471] [Q&A]
I'm just wondering how can I test this on Thunder Client or Postman
I'm getting this error.

![image](https://user-images.githubusercontent.com/20082606/184909831-1f92045a-94f3-4731-b233-6d6b8feadbde.png)

**Comment by sachinraja** [comment]
tRPC has a [special RPC format](https://trpc.io/docs/rpc) so REST clients won't work that well (especially for queries). You can try https://github.com/sachinraja/trpc-playground.
